adoc/deployment-bootstrap.adoc

```diff
@@ -3,13 +3,92 @@
 
 === Cluster Deployment
 
+==== Cluster initialization
 . Initialize the cluster on the deployed machines.
 As `--control-plane` enter the IP/FQDN of your load balancer. If you do not use a load balancer use your first master node.
 +
 ----
 skuba cluster init --control-plane <LB IP/FQDN> my-cluster
 ----
 `cluster init` generates the folder named `my-cluster` and initializes the directory that will hold the configuration for the cluster.
+
+==== Cluster configuration
+
+At this point you can perform some configurations before bootstrapping the cluster.
+
+===== Prevent nodes running special workloads from being rebooted
+
+. Open the `kured` deployment in `my-cluster/addons/kured/kured.yaml`
+. Adapt the `DaemonSet` by adding one of the following flags to the `command` section of the `kured` container:
++
+----
+---
+apiVersion: apps/v1
+kind: DaemonSet
+...
+spec:
+  ...
+    ...
+      ...
+      containers:
+        ...
+          command:
+            - /usr/bin/kured
+            - --blocking-pod-selector=name=<NAME OF POD>
+----
+
+You can add any key/value labels to this selector:
+----
+--blocking-pod-selector=<LABEL KEY 1>=<LABLE VALUE 1>,<LABEL KEY 2>=<LABEL VALUE 2>
+----
```

Comment:
@MaximilianMeister This may sounds stupid but I would add the command `kubectl apply -f my-cluster/addons/kured/kured.yaml` so the user knows that it has to be done with `kubectl` and not with `skuba`

Reply:
this happens before the bootstrap, so `skuba` will do the actual apply during bootstrap. here we just adapt the manifest before skuba reads it

Reply:
@MaximilianMeister yes but on a running cluster he would have to do it like that. 

What happens if he `kubectl edit` the deployment and then add a new node ?

Reply:
you can't do `kubectl apply` from the init folder, as there are some variables (e.g. image name) in the yaml that are not expanded (skuba does that during bootstrap)

> What happens if he `kubectl edit` the deployment and then add a new node ?

@ereslibre the join action will apply kured from which manifest?

Reply:
> @ereslibre the join action will apply kured from which manifest?

Kured is a daemonset and when joining a new node we are not doing anything special with it.

A different story is when upgrading the platform, but that's out of scope right now :)

Reply:
ok thanks. so kubelet will create the deployment on the new node as it is in the current state in `etcd` - so it should be fine

Reply:
@MaximilianMeister it works to update the local file and then run `kubectl apply`, I find this is a better way than using `kubectl edit` as the changes are "consistent" between the initial deployed file and the running `ds`

Reply:
ok makes sense, let's stick to one way of doing it. updated to `kubectl apply`